docs/deployments/packages/delta-compression-for-package-transfers.md

```diff
@@ -16,12 +16,11 @@ A typical scenario in Octopus Deploy is frequent deployments of small changes to
 
 ## A package deployment in Octopus Deploy now looks something like this {#Deltacompressionforpackagetransfers-ApackagedeploymentinOctopusDeploynowlookssomethinglikethis}
 
-1. Find nearest previous versions of the package on the machine by calling [Calamari](https://octopus.com/blog/calamari).
-2. If a previous version of a package is found and the same package (matching PackageId, Version and file hash) exist on the Octopus Server.
-3. Create a signature file for the nearest package found.
-4. Build the delta between the previous package (using the above signature file) and the new package.
-5. Upload delta file to Tentacle and call Calamari to apply the delta file to the package found in the previous step.
-6. If no previous version of the package was found, we upload the full package instead.
+1. Find the nearest previous version of the package on the target machine by calling [Calamari](https://octopus.com/blog/calamari).
```

Comment:
It is actually version**s**. We ask Calamari to grab a bunch of versions and then look for a package match on the server.